config.toml

```diff
@@ -2,6 +2,7 @@ baseURL = "https://keda.sh"
 languageCode = "en-us"
 title = "KEDA"
 enableGitInfo = true
+enableRobotsTXT = true
```

Comment:
Are you claiming that this affects accessibility ratings? Revert this change.

Reply:
Hi chalin, this was for improving SEO to appear in search results and search engine indexing.

Reply:
should i keep this or remove it?

Reply:
Drop it for this PR. I'm willing to be convinced otherwise, but as far as I know this buys us nothing.

Reply:
ok i'll revert this